src/main/java/io/candydoc/ddd/shared_kernel/SharedKernelInteractionStrategy.java

```diff
@@ -0,0 +1,14 @@
+package io.candydoc.ddd.shared_kernel;
+
+import io.candydoc.ddd.Event;
+import io.candydoc.ddd.interaction.InteractionStrategy;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class SharedKernelInteractionStrategy implements InteractionStrategy<SharedKernel> {
+
+  public List<Event> checkInteractions(SharedKernel concept) {
+    return List.of();
```

Comment:
Here, you should check the interactions with bounded contexts and shared kernels. Same in BoundedContextInteractionStrategy.